src/include/migraphx/match/gelu_tanh.hpp

```diff
@@ -36,31 +36,43 @@ template <class F>
 struct gelu_tanh_matcher
 {
     F f;
-    auto pow_fn() const { return f("pow")(used_once(), arg(1)(has_value(3.0f))); }
 
+    /// x ^ 3
+    auto pow_fn() const { return f("pow")(used_once(), arg(1)(has_value(3.0))); }
+
+    /// tanh( sqrt(2/M_PI) * (x + 0.044715 * x ^ 3 )
     auto tanh_fn() const
     {
-        return f("tanh")(
-            used_once(),
-            arg(0)(f("mul")(either_arg(0, 1)(has_value(sqrt(M_2_PI), 1e-3),
-                                             f("add")(any_arg(0, 1)(f("mul")(either_arg(0, 1)(
-                                                 has_value(0.044715f), pow_fn()))))))));
+        auto mul_const_pow    = f("mul")(either_arg(0, 1)(has_value(0.044715), pow_fn()));
+        auto add_any_mul   = f("add")(any_arg(0, 1)(mul_const_pow));
+        auto mul_SQRT2RPI_add = f("mul")(either_arg(0, 1)(has_value(sqrt(M_2_PI)), add_any_mul));
```

Comment:
`mul_SQRT2RPI_add` doesn't match the casing style. It should be `mul_sqrt2rpi_add`. I think clang tidy is complaining about that.